src/js/select2/i18n/fa.js

```diff
@@ -0,0 +1,38 @@
+/* jshint -W100 */
+/*jslint maxlen: 130 */
```

Comment:
I'd rather avoid squashing this warning considering it's quite easy to break up strings in JavaScript.